system/doc/reference_manual/expressions.xml

```diff
@@ -1348,6 +1348,27 @@ handle_call(change, From, #{ state := start } = S) ->
   <section>
     <marker id="bit_syntax"></marker>
     <title>Bit Syntax Expressions</title>
+    <p>The bit syntax operates on <em>bit strings</em>.</p>
+    <p>
+      A <em>bit string</em> is a sequence of bits stored in
+      a sequence of (8-bit) bytes in memory address order.
+      The bits are stored in big-endian format in the byte sequence
+      so the first bit is the higest bit of the first byte.
```

Comment:
I don't think "big-endian" is normally used to refer to bit order.
Suggestion:
_A **bit string** is a sequence of bits. The bits are stored as
a sequence of (8-bit) bytes in memory address order.
Within each byte the bits are stored from highest to lowest bit. That is,
the first bit of a bit string is the highest bit of the first byte._

Reply:
Isn't the usual term "most/least significant" bit?

Reply:
Will do